### PR TITLE
Update textx example path in README.md

### PR DESCRIPTION
**Pull request by LucaKro**
Example should now run out of the box if you just follow the commands in the readme from top to bottom

### CODE REVIEW

**Review by argenos — COMMENTED** [review]
It should be possible to use the `textx generate` command from anywhere, so _technically_ the path isn't wrong.

**Comment by LucaKro** [comment]
> It should be possible to use the `textx generate` command from anywhere, so _technically_ the path isn't wrong.

I mean I did not say the path was wrong, just that it doesn't work out of the box if you just follow the readme from top to bottom. Imo this is just a small quality of life adjustment, and i don't see the benefit of keeping the original over this version.

But also its not really that important, so if you want to keep the original, feel free to close this pr instead ^^

**Review by argenos — CHANGES_REQUESTED** [review]
Gotcha! In that case, please change the blender calls above to use `src/....` instead

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ An example model for a building is available [here](models/examples/hospital.flo
 
 
 ```
+cd src
```
Comment (argenos) [inline README.md:99]: Sorry for nitpicking. Don't cd into src, use python the same way you added in the lines above by calling Python and passing src...

Remove the ../ from the models path

File: README.md [hunk 2]
```diff
@@ -125,7 +126,7 @@ textx generate <floorplan model> --target json-ld --output-path <output path>
 For example: 
 
 ```
-textx generate models/examples/brsu_building_c_with_doorways.floorplan --target json-ld --output-path .
+textx generate ../models/examples/brsu_building_c_with_doorways.floorplan --target json-ld --output-path .
```
Comment (argenos) [inline README.md:129]: Remove the ../ since we should not cd into src in the lines above

**Review by argenos — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (LucaKro) [reply README.md]: paths should be fixed now, if i understood your request correctly

**Review by argenos — APPROVED** [review]

**Pull request merged**